src/modules/user/user.service.ts

```diff
@@ -31,8 +31,15 @@ export class UserService extends BaseService<UserEntity, UserRepository> {
         return this._findById(id);
     }
 
-    getInactiveUsers(filter: iPaginationOption): Promise<PaginationResponse<UserEntity>> {
-        return this.repository.getInactiveUsers(filter.page, filter.limit);
+    async getInactiveUsers(filter: iPaginationOption): Promise<PaginationResponse<UserEntity>> {
+        // const qb = this.repository.createQueryBuilder().where('is_active = :active', { active: true });
+        // return this._iPaginate(qb, filter.page, filter.limit);
+        // const repo = this.repository.manager.getRepository(UserEntity);
+        // const qb2 = repo.createQueryBuilder().where('is_active = :active', { active: true });
+        // return this._iPaginate(qb2, filter.page, filter.limit);
+
+        const rs = await this.repository.getInactiveUsers(filter.page, filter.limit);
```

Comment:
Change variable name `rs ` because I don't understand this variable describes for what